lib/modules/apostrophe-workflow-pages/index.js

```diff
@@ -179,9 +183,21 @@ module.exports = {
               target: info.target.workflowGuid,
               position: info.position
             },
-            ...moved.workflowLocale.includes('-draft') && { workflowMovedIsNew: true }
+            ...isDraft && { workflowMovedIsNew: true }
           }
-        }, callback);
+        }, (err) => {
+          if (err || !isDraft || !syncPagesMoves) {
+            return callback(err || null);
+          }
+
+          self.apos.workflow.getDraftAndLive(req, moved._id, {}, (err, draft, live) => {
+            if (err) {
+              return callback(err);
+            }
+
+            return self.apos.workflow.repeatMove(req, draft, live, callback);
```

Comment:
There is a permissions problem here. The user might only have permission to take actions on the drafts. In which case this will fail when it tries to commit the move to live.

So we will have to add an options parameter to `repeatMove` so we can pass in `{ permissions: false }` and make sure we pass that along to the methods it invokes.

For bc `repeatMove` will also have to detect when it is called with only four arguments and still do what it did before.

Note that this means users who cannot commit can still affect the live page tree. That is an unavoidable consequence of this feature I think.

The other option would be to not allow users who cannot commit to move pages at all.

Reply:
You pointed out an important thing here.
Personally, I think that I would not allow users who cannot commit to move pages at all. Since the move, as you said above, is like a minimal commit, so they are no reasons that a person who cannot commit could commit just some properties.

Reply:
As we discussed, If a user can only edit pages, and `autoCommitPagesMoves` is true, this user cannot move pages.